<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDeviceInfoClass</class>
 <widget class="QDialog" name="EditDeviceInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>166</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>457</width>
    <height>166</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>457</width>
    <height>166</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EditDeviceInfo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="4,1">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="device_name_label">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string> Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="device_name_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="topMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="type_name_label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>*Type</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="mx_radio_btn">
             <property name="text">
              <string>M/X/G Series</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="gr_radio_btn">
             <property name="text">
              <string>GR Series</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="syn_radio_btn">
             <property name="text">
              <string>Sync Controller</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ip_layout">
       <item>
        <widget class="QLabel" name="ip_name_label">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>*IP</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/refaapp.qrc"/>
 </resources>
 <connections/>
</ui>
